src/pages/index.tsx

```diff
@@ -48,7 +47,7 @@ export default function Home() {
   const fetchingNext = voteMutation.isLoading || isLoading;
 
   return (
-    <div className="h-screen w-screen flex flex-col justify-between items-center relative">
```

Comment:
This can also be addressed by simply adding a border to this element. I think the following classes can be added to the existing ones: `border-8 border-transparent border-solid`